src/_cffi_src/openssl/bio.py

```diff
@@ -57,4 +59,13 @@
     free(ptr);
 }
 #endif
+
+#if CRYPTOGRAPHY_IS_BORINGSSL
+static const long Cryptography_HAS_BIO_DGRAM = 0;
+
+BIO *(*BIO_new_dgram)(int, int) = NULL;
```

Comment:
Do we actually need these? They were landed in https://github.com/pyca/cryptography/pull/4089 for DTLS support, but https://github.com/pyca/pyopenssl/pull/1026 added DTLS support to pyOpenSSL and explicitly does not use the dgram BIOs.

Reply:
I'd be delgihted to delete them! Separate PR incoming.